src/components/admin/ImageInput/ImageInput.tsx

```diff
@@ -40,10 +40,11 @@ export interface ImageInputProps {
   setValue: UseFormSetValue<AnyForm>;
   register: UseFormRegister<AnyForm>;
   nameWithUnderscore?: boolean;
-  text?: string;
   subtext?: string;
   // isInputHidden?: boolean;
   variant?: ImageInputVariant;
+  label?: string;
+  buttonLabel?: string;
```

Comment:
This is hard to follow. If there is a single one, `label` is good enough, but with tho, it's easier to not have to remember what the short one stands for, with something like `inputLabel` and `buttonLabel`

Reply:
done, [3469e91](https://github.com/sparkletown/sparkle/pull/2944/commits/3469e91693f840fa4d04232bc8c99b452b8f6b6d)